src/components/organize/tasks/TaskAssigneesList/index.tsx

```diff
@@ -0,0 +1,51 @@
+import dayjs from 'dayjs';
+import { useIntl } from 'react-intl';
+import { Box, Card, Grid } from '@material-ui/core';
+
+import TaskStatusSubtitle from './TaskStatusSubtitle';
+import ZetkinPerson from 'components/ZetkinPerson';
+import ZetkinSection from 'components/ZetkinSection';
+import { ASSIGNED_STATUS, ZetkinAssignedTask } from 'types/tasks';
+
+const TaskAssigneesList: React.FunctionComponent<{ assignedTasks: ZetkinAssignedTask[] }> = ({ assignedTasks }) => {
+    const intl = useIntl();
+
+    const sortedAssignedTasks = assignedTasks.sort((first, second) => {
+        if (first.status === ASSIGNED_STATUS.COMPLETED && second.status !== ASSIGNED_STATUS.COMPLETED) return -1;
+        if (first.status !== ASSIGNED_STATUS.COMPLETED && second.status === ASSIGNED_STATUS.COMPLETED) return 1;
+        if (first.status === ASSIGNED_STATUS.ASSIGNED && second.status === ASSIGNED_STATUS.IGNORED) return 1;
+        if (first.status === ASSIGNED_STATUS.IGNORED && second.status === ASSIGNED_STATUS.ASSIGNED) return -1;
+
+        const firstDate = first.completed ? dayjs(first.completed) : dayjs(first.assigned);
+        const secondDate = second.completed ? dayjs(second.completed) : dayjs(second.assigned);
+
+        if (firstDate.isBefore(secondDate)) return 1;
+        if (firstDate.isAfter(secondDate)) return -1;
+
+        return 0;
```

Comment:
This could also be simplified (different logic, same result) as `return secondDate - firstDate;`

Reply:
Cool!! thank you!